package.json

```diff
@@ -64,14 +64,17 @@
     "bitcore-lib": "^0.13.7",
     "bitcore-message": "^1.0.1",
     "body-parser": "^1.13.3",
+    "compression": "^1.6.1",
     "lodash": "^2.4.1",
+    "lru-cache": "^4.0.1",
+    "morgan": "^1.7.0",
     "request": "^2.64.0"
   },
   "devDependencies": {
-    "chai": "*",
-    "mocha": "~1.16.2",
+    "chai": "^3.5.0",
+    "mocha": "^2.4.5",
     "proxyquire": "^1.7.2",
-    "should": "^2.1.1",
+    "should": "^8.3.1",
```

Comment:
Yes mostly, the API is somewhat different though. I think should is used here because it displays large diffs better.